src/keys/utils.ts

```diff
@@ -176,11 +177,10 @@ function publicKeyToFingerprintBytes(
   return fString;
 }
 
-function publicKeyToFingerprint(publicKey: PublicKey): PublicKeyFingerprint {
+function publicKeyToNodeId(publicKey: PublicKey): NodeId {
```

Comment:
I suggest that since `NodeId` is defined in `nodes`, this function should instead be in `nodes/utils`.

Then the `KeyManager` can import `nodesUtils` and call that function.

Do we need `publicKeyToFingerprintBytes` function anymore too?

Reply:
Actually disregard this comment, mainly because it uses alot of utilities inside `keysUtils` already. So conversion to `NodeId` stays here.

However still `publicKeyToFingerprintBytes`, if it is not used elsewhere, it could also be just embedded into this function.

Reply:
Removed `publicKeyToFingerprintBytes` and in-lined it into `publicKeyToNodeId`